Jenkinsfile_nightly

```diff
@@ -37,9 +37,14 @@ withNightlyPipeline("nodejs", product, component) {
 
     before('DependencyCheckNightly') {sh 'yarn test:audit'}
     enableCrossBrowserTest(60)
+    enableFortifyScan('sscs-aat')
```

Comment:
Why we have 'sscs-aat' here?

Reply:
By default, it is taking keyvault name as vaultName = "${var.product}-${var.env}" which is sscs-tribunals-aat
The fortify process looks for sscs-aat keyvault

Reply:
I have modified to 'sscs-${env}'